code/modules/jobs/job_types/roguetown/adventurer/types/combat/ranger.dm

```diff
@@ -129,3 +130,65 @@
+			H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/bows, 1, TRUE) // Base skill, if not wanted, pick another weapon.
+			H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/axes, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 2, TRUE) // On par with battlemaster.
+			H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/swimming, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/climbing, 3, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 2, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/craft/tanning, 1, TRUE) // Still a ranger, nerfed. Want more? Go do it yourself, buddy.
+			H.mind.adjust_skillrank(/datum/skill/labor/butchering, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/craft/cooking, 1, TRUE) // Won't really equate to much.
+			H.mind.adjust_skillrank(/datum/skill/misc/tracking, 3, TRUE)
+			ADD_TRAIT(H, TRAIT_OUTDOORSMAN, TRAIT_GENERIC)
+			var/weapons = list("Recurve Bow","Billhook","Sling")
+			var/weapon_choice = input("Choose your weapon.", "TAKE UP ARMS") as anything in weapons
+			switch(weapon_choice)
+				if("Recurve Bow")
+					H.mind.adjust_skillrank(/datum/skill/combat/bows, 2, TRUE)
+					backr = /obj/item/gun/ballistic/revolver/grenadelauncher/bow/recurve
+					beltl = /obj/item/quiver/arrows
+				if("Billhook") // Debatable here, but we love variety.
+					H.mind.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
+					r_hand = /obj/item/rogueweapon/spear/billhook
+					backr = /obj/item/gwstrap
+				if("Sling")
+					H.mind.adjust_skillrank(/datum/skill/combat/slings, 3, TRUE)
+					beltl = /obj/item/quiver/sling/iron
+					r_hand = /obj/item/gun/ballistic/revolver/grenadelauncher/sling
+			var/armors = list("Light Armor","Medium Armor")
+			var/armor_choice = input("Choose your armor.", "TAKE UP ARMS") as anything in armors
+			switch(armor_choice)
+				if("Light Armor")
+					armor = /obj/item/clothing/suit/roguetown/armor/leather/hide
+					pants = /obj/item/clothing/under/roguetown/heavy_leather_pants
+					gloves = /obj/item/clothing/gloves/roguetown/fingerless_leather
+					ADD_TRAIT(H, TRAIT_DODGEEXPERT, TRAIT_GENERIC)
+				if("Medium Armor")
+					armor = /obj/item/clothing/suit/roguetown/armor/chainmail/iron
+					pants = /obj/item/clothing/under/roguetown/chainlegs/iron
+					gloves = /obj/item/clothing/gloves/roguetown/chain/iron
+					ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
+					H.set_blindness(0)
+			H.change_stat("perception", 2) // Look far, but not too far.
+			H.change_stat("speed", 1)
+			H.change_stat("endurance", 1)
+			H.change_stat("strength", 1)
+			H.change_stat("intelligence", 1) // Adaptive to their surroundings.
```

Comment:
This is weighted to 8. (SPD / STR at 2, rest at 1)
Adventurers are meant to be weighted at 7, so this should be adjusted. Up to you how, though!